keps/sig-autoscaling/2702-graduate-hpa-api-to-GA/README.md

```diff
@@ -0,0 +1,426 @@
+# Graduate v2beta2 Autoscaling API to GA
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Implementation Details](#implementation-details)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Renames](#renames)
+  - [Test Plan](#test-plan)
+  - [Graduation Criteria](#graduation-criteria) 
+  - [Version Skew Strategy](#version-skew-strategy)
+  - [Upgrade/Downgrade Strategy](#upgradedowngrade-strategy)
+- [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+  - [Feature Enablement and Rollback](#feature-enablement-and-rollback)
+  - [Rollout, Upgrade and Rollback Planning](#rollout-upgrade-and-rollback-planning)
+  - [Monitoring Requirements](#monitoring-requirements)
+  - [Dependencies](#dependencies)
+  - [Scalability](#scalability)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Summary
+
+This document outlines required steps to graduate autoscaling v2beta2 API to GA.
+
+## Motivation
```

Comment:
"The HPA v2 series APIs were first introduced in November, 2016 (5 years ago).  The primary feature of the v2 series is adding support for multiple and custom metrics.  The structure was improved slightly in the v2beta2 API which became available in May 2018 and has remained largely unchanged since then.  The v2beta2 API has been used extensively and informally treated as stable.  The motivation for this KEP is to push it over the line to make it formally so.